i18n/bg/metadata.yml

```
@@ -0,0 +1,82 @@
+title: Децентрализиран протокол за възнаграждение на екосистемата с отворен код
+abstract: |-
+Създаването на отворен, публичен и стабилен регистър за целия софтуер с отворен код 
```

```suggestion
  Създаването на отворен, публичен и стабилен регистър за целия софтуер с отворен код 
```

Please properly indent as this is YAML and correct indents are important